app/core/download.py

```diff
@@ -274,4 +275,38 @@ def convert_dump_to_csv(df: pd.DataFrame):
 def generate_data_dump_as_csv(ingest_cycle_start: str, db=Depends(get_db)):
     df = get_whole_database_dump(ingest_cycle_start, db)
     csv = convert_dump_to_csv(df)
+    csv.seek(0)
     return csv
+
+
+def generate_data_dump_readme(ingest_cycle_start: str):
+    file_buffer = StringIO(
+        "Thank you for downloading the full document dataset from Climate Policy Radar "
+        "and Climate Change Laws of the World!"
+        "\n\n"
+        "For more information including our data dictionary, methodology and "
+        "information about how to cite us, visit "
+        "\n"
+        "https://climatepolicyradar.notion.site/Readme-for-document-data-download-f2d55b7e238941b59559b9b1c4cc52c5."
```

Comment:
That full stop at the end of the line could be problematic for the link:
```suggestion
        "https://climatepolicyradar.notion.site/Readme-for-document-data-download-f2d55b7e238941b59559b9b1c4cc52c5"
```